nuxt.config.js

```diff
@@ -98,7 +98,10 @@ export default {
     // https://go.nuxtjs.dev/vuetify
     "@nuxtjs/vuetify",
   ],
-
+  env: {
```

Comment:
this is how to pass current env to the client side.

Reply:
Aha, moment; was wondering about this.